lib/install/opsuser/opsuser.go

```diff
@@ -65,8 +65,8 @@ func GrantDCReadOnlyPerms(ctx context.Context, session *session.Session, configS
 	return err
 }
 
-func GrantOpsUserPerms(ctx context.Context, client *vim25.Client, configSpec *config.VirtualContainerHostConfigSpec) error {
-	mgr := NewRBACManager(ctx, client, nil, &OpsuserRBACConf, configSpec)
+func GrantOpsUserPerms(ctx context.Context, session *session.Session, configSpec *config.VirtualContainerHostConfigSpec) error {
+	mgr := NewRBACManager(ctx, session.Vim25(), session, &OpsuserRBACConf, configSpec)
```

Comment:
I am just curious why this manager needs both the sessions and the vim25 reference to the session in the same header. If it has the session then it can get the vim25 client inside of the `NewRBACManager` call. Are we expecting a potentially different client than the session we pass?? that seems dangerous to me.

Reply:
Good call.

> Are we expecting a potentially different client than the session we pass?

Definitely not - fixed.